projects/sbb-esta/angular-business/table/table.md

```diff
@@ -0,0 +1,53 @@
+The table module allows the user to display structured data within a table.
```

Comment:
The documentation is a bit short. Maybe compare with https://material.angular.io/components/table/overview. (It is not my expectation, that it should be documented to the same extent. But maybe add documentation for the most important features, like the data source.)

Reply:
I double-checked all of the features described in the Material documentation and updated ours to match the actual parts of it that we provide too.